tests/Cluster.py

```diff
@@ -65,6 +66,12 @@ def __init__(self, walletd=False, localCluster=True, host="localhost", port=8888
         defproducerbPrvtKey: Defproducerb account private key
         """
         self.accounts={}
+        self.clusterID=clusterID
+        if clusterID != 0:
```

Comment:
This and the related lines below would have to also be changed to reflect the new port assignment scheme I proposed above, as well as to not treat a `clusterID` of 0 as a special case and also to ensure 5 digits are always included in the name of the bios boot shell script.